packages/template-retail-react-app/app/components/drawer-menu/index.test.js

```diff
@@ -8,47 +8,16 @@ import React from 'react'
 import DrawerMenu from './index'
 import {renderWithProviders} from '../../utils/test-utils'
 
-const mockRoot = {
-    id: 't1',
-    name: 'Test One',
-    categories: [
-        {
-            id: 't1-1',
-            name: 'Test One One'
-        },
-        {
-            id: 't1-2',
-            name: 'Test One Two',
-            categories: [
-                {
-                    id: 't1-2-1',
-                    name: 'Test One Two One'
-                },
-                {
-                    id: 't1-2-2',
-                    name: 'Test One Two Two'
-                }
-            ]
-        }
-    ]
-}
+describe('DrawerMenu', () => {
+    test('Renders DrawerMenu without errors', () => {
+        renderWithProviders(<DrawerMenu isOpen={true} />)
 
-test('Renders DrawerMenu with root', () => {
-    renderWithProviders(<DrawerMenu root={mockRoot} isOpen={true} />)
+        const drawer = document.querySelector('.chakra-portal')
+        const accordion = document.querySelector('.chakra-accordion')
+        const socialIcons = document.querySelector('.sf-social-icons')
 
-    const drawer = document.querySelector('.chakra-portal')
-    const accordion = document.querySelector('.chakra-accordion')
-    const socialIcons = document.querySelector('.sf-social-icons')
-
-    expect(drawer).toBeInTheDocument()
-    expect(accordion).toBeInTheDocument()
-    expect(socialIcons).toBeInTheDocument()
-})
-
-test('Renders DrawerMenu Spinner without root', () => {
-    renderWithProviders(<DrawerMenu isOpen={true} />)
-
-    const spinner = document.querySelector('.chakra-spinner')
-
-    expect(spinner).toBeInTheDocument()
```

Comment:
Do we want to cut the spinner? Granted first-level categories get fetch server-side, but the same was true when the spinner was in place before. Testing this will be a bit tougher, but the flip side is this will cause a loss of test coverage %.   Tough call 🤔

Reply:
I think the goal in this test is to make sure that a failed server-side API fetch in `_app/index.jsx` yields a spinner on the client side, which is a good test

Reply:
Added more test cases for both list-menu and drawer-menu when there are no root categories provided